File: Python/NarcissisticNumber/MySolution.py
def narcissistic( value ):
    tmp = 0
    for i in str(value):
        tmp += int(i)**len(str(value))
        
    return True if tmp == value else False

# Original kata: https://www.codewars.com/kata/5287e858c6b5a9678200083c
# My solution: https://www.codewars.com/kata/reviews/55db4cb62ede87b90600007e/groups/675867adf5291907e9265570

File: Rust/PlayBanjo/MySolution.rs
fn are_you_playing_banjo(name: &str) -> String {
    if name.to_uppercase().starts_with('R') {
        format!("{} plays banjo", name)
    } else {
        format!("{} does not play banjo", name)
    }
}

// Original kata: https://www.codewars.com/kata/53af2b8861023f1d88000832
// My solution: https://www.codewars.com/kata/reviews/5f84898f6ddaa3000166fbea/groups/5fd522bfe7f5d20001b0d699
